data/data-pipeline/data_pipeline/etl/base.py

```diff
@@ -35,7 +35,7 @@ class ExtractTransformLoad:
     # TODO: investigate. Census says there are only 217,740 CBGs in the US. This might be from CBGs at different time periods.
     EXPECTED_MAX_CENSUS_BLOCK_GROUPS: int = 250000
     # TODO: investigate. Census says there are only 73,057 tracts in the US. This might be from tracts at different time periods.
```

Comment:
Do we have any more insight here to update this comment?

Reply:
Honestly no, I don't know where those extra tracts are coming from right now. I created this ticket for the backlog: https://github.com/usds/justice40-tool/issues/964. 

Turns out the problem is less bad than we thought -- including the island areas and PR, census says there should be 74,134 tracts, so we have 26 bonus tracts.